src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+export const Tabs = () => {
+  const { tabId = 0 } = useParams();
```

Comment:
```suggestion
  const { tabId } = useParams();
```

Why do you need a default value here? Just remember to handle undefined value.